services/common/src/components/projects/ProjectDocumentsTab.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { FC, useEffect, useState } from "react";
-import { useDispatch, useSelector } from "react-redux";
+import { useSelector } from "react-redux";
```

Comment:
Any reason we can't use the `useAppSelector` here?

Reply:
Updated 👍